pyspark/graphar_pyspark/graph.py

```diff
@@ -0,0 +1,215 @@
+"""
+Copyright 2022-2023 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from py4j.java_gateway import JavaObject
+from pyspark.sql import DataFrame
+
+from graphar_pyspark import GraphArSession
+from graphar_pyspark.enums import FileType
+from graphar_pyspark.info import GraphInfo
+
+
+class GraphReader:
+    """The helper object for reading graph through the definitions of graph info."""
+
+    @staticmethod
+    def _jvm2py(
+        jvm_result: tuple[
+            dict[str, JavaObject], dict[tuple[str, str, str], dict[str, JavaObject]]
+        ],
+    ) -> tuple[dict[str, DataFrame], dict[tuple[str, str, str], dict[str, DataFrame]]]:
```

Comment:
Minor: this type: `tuple[dict[str, DataFrame], dict[tuple[str, str, str], dict[str, DataFrame]]]` appears in a few places and it seems that we actually refer to the same thing. Maybe define
- an alias, like
```python
MyReturnTypeWhatever = tuple[dict[str, DataFrame], dict[tuple[str, str, str], dict[str, DataFrame]]]
```
and refer to that, or
- define a tiny data structure so that instead of a `tuple` we have a simple class (maybe `NamedTuple` or `dataclass`) that wraps the content?

Reply:
I like the option with creating a return type. @acezen what do you think? It won't directly follow scala api, but having a dataclass with fields, that can be accessed by autocomplete is better, than working with plain dicts. It will be just a very small dataclass without method (or maybe with one, that contain the logic of `_jvm2py` method and create an instance of the class from scala output)

Reply:
Added a return-object